dep_tools/loaders.py

```diff
@@ -130,9 +130,10 @@ def _get_items(self, area):
 
 
 class OdcLoaderMixin:
-    def __init__(self, odc_load_kwargs, **kwargs):
+    def __init__(self, odc_load_kwargs, nodata_value: float | None = None, **kwargs):
```

Comment:
```suggestion
    def __init__(self, odc_load_kwargs, nodata_value: Union[float, None] = None, **kwargs):
```

I think the typing way to do this is like this... need the import `from typing import Union`

Reply:
Prior to 3.10 yes, if we're using 3.10 this is OK. https://peps.python.org/pep-0604/

Reply:
Ah, ok!

Reply:
`docker run --rm ghcr.io/digitalearthpacific/dep-mangroves python --version`
> Python 3.10.12